pkg/util/v1beta1/env/env.go

```diff
@@ -16,11 +16,26 @@ limitations under the License.
 
 package env
 
-import "os"
+import (
+	"k8s.io/klog"
+	"os"
+	"strconv"
+)
 
 func GetEnvOrDefault(key string, fallback string) string {
 	if value, ok := os.LookupEnv(key); ok {
 		return value
 	}
 	return fallback
 }
+
+func GetBoolEnvOrDefault(key string, fallback bool) bool {
```

Comment:
Do we want to introduce this new function to get `bool` env var ?
Previously, even for bool variables, we compare them with strings: https://github.com/kubeflow/katib/blob/f4c8861c810cc3088beebe9578932151850eafd1/pkg/ui/v1beta1/authzn.go#L47
cc @tenzen-y @johnugeorge

Reply:
I don't think that we want to have this function.